### DISCUSSION TITLE
Difficulty Deleting User Avatar Folders in Supabase

### DISCUSSION THREAD

**Discussion by liwa-dev** [supabase/supabase#20066] [Questions]
I'm facing an issue while attempting to delete user avatar folders stored in Supabase. Specifically, I encounter difficulties when trying to remove folders with names like "user_{username}_{email}/avatar.{fileType (img, png, etc.)}". Despite exploring the documentation and various approaches, I've been unsuccessful in programmatically deleting these folders.

I've tried different methods, including converting folder names to strings and reviewing Supabase policies. However, none of these approaches have resolved the issue. I would greatly appreciate guidance on the correct procedures or necessary policy settings within Supabase that would enable the deletion of these folders.

Moreover, while I'm able to "UPLOAD" the file and execute "SELECT" queries successfully, I'm unable to perform the "DELETE" action. Even though I've granted the necessary permissions by setting the Policy to delete, attempting to delete a file results in an error where the variable "data" appears as undefined.

Here's a snippet of the code I'm using:
```js
       //getImg[0].avatar_url gives -> "user_{username}_{username}/avatar.{fileType}" from select query table
        const imgName = getImg[0].avatar_url;
        //remove old file 
        const { data: dataOldFile, error:removeOldFile } = users.storage
        .from("pictures")
        .remove([imgName]);
        console.log('Detected file successfully delete it in storage',dataOldFile); // <-- i get undefined here in "dataOldFile"
```

**Comment by GaryAustin1** [comment]
What are your delete and select policies?
Can you show an example of an actual path string?
Any error in the error return?
